bokehjs/src/coffee/widget/vbox.coffee

```diff
@@ -23,9 +24,20 @@ class VBoxView extends ContinuumView
     width = @mget("width")
     if width? then @$el.css(width: width + "px")
     height = @mget("height")
-    if height? then @$el.css(height: height + "px")
+    if height?
+      @$el.css(height: height + "px")
+      t = (children.length-2)*2
+      spacer_height = height/(children.length*2)
+    else
+      spacer_height = 20
+
+    spacer = '<div class="bk-vbox-spacer" style="height: ' + spacer_height + 'px;"></div>';
```

Comment:
This should use string interpolation, or better `$('<div>').addClass('bk-vbox-spacer').css({height: spacer_height})`.